### PR TITLE
Add office hour teaser

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/15720

This replaces the textual office hours link with a graphical banner.

The block only renders until March 3, 2021, to avoid us forgetting about outdated content.

## Preview

image

image

### CODE REVIEW

**Comment by marians** [comment]
Updated the link

File: src/layouts/partials/help_feedback.html [hunk 1]
```diff
@@ -1,6 +1,12 @@
 {{ with .File }}
 <div class="feedback well">
   <h5><i class="fa fa-help-outline"></i> Need help, got feedback?</h5>
-  <p>We listen to your Slack support channel. You can also reach us at <a href="mailto:support@giantswarm.io">support@giantswarm.io</a>. To continue the conversation, join us at our <a href="https://swarm.ly/37lDbAj" target="_blank" rel="noopener">monthly Office Hour</a>. And of course, we welcome your <a title="open this page in GitHub" href="https://github.com/giantswarm/docs/blob/master/src/content/{{ .Path }}" target="_blank" rel="noopener noreferrer">pull requests</a>!</p>
+  <p>We listen to your Slack support channel. You can also reach us at <a href="mailto:support@giantswarm.io">support@giantswarm.io</a>. And of course, we welcome your <a title="open this page in GitHub" href="https://github.com/giantswarm/docs/blob/master/src/content/{{ .Path }}" target="_blank" rel="noopener noreferrer">pull requests</a>!</p>
+</div>
+{{ end }}
+
+{{ if le (now.Format "2006-01-02") "2021-03-03" }}
```
Comment (stone-z) [inline src/layouts/partials/help_feedback.html:8]: Just curious -- is `2006-01-02` a magic value or something?

**Review by stone-z — APPROVED** [review]
Cool!

File: src/layouts/partials/help_feedback.html (hunk 1 above)
Comment (marians) [reply src/layouts/partials/help_feedback.html:8]: That's just Go's magic reference time form formatting. https://golang.org/pkg/time/#pkg-constants

**Pull request merged**